Examples/AferoSwiftSDKTests/Core/RESTClientAPI/AferoAPIClient+AccountTests.swift

```diff
@@ -48,7 +48,7 @@ class APIClientAccountSpec: QuickSpec {
                     
                     stub(condition: isPath("/v1/users/me") && isMethodGET()) {
                         request in
-                        return OHHTTPStubsResponse(jsonObject: janeDoeUser.JSONDict!, statusCode: 200, headers: nil)
+                        return OHHTTPStubs.HTTPStubsResponse(jsonObject: janeDoeUser.JSONDict!, statusCode: 200, headers: nil)
```

Comment:
Don't have the source in front of me, but confused... is it correct that it's `OHTTPStubs....` here, and `HTTPStubs....` on line 36 above?

Reply:
This is kind of weird, they are both in the same pod, if I change it it won't recognize it